files/zh-cn/web/javascript/reference/statements/async_function/index.html

```diff
@@ -234,7 +234,7 @@ <h4 id="asyncawait和Promisethen对比以及错误处理"><code>async</code>/<co
  虽然<code>parallelPromise</code>函数示例看起来很简单，但它根本不会处理错误！ 这样做需要一个类似于<code>return </code><code>Promise.all([])</code>处理方式。</p>
 </div>
 
-<h3 id="使用async函数重写_promise_链"><code><font face="x-locale-heading-primary, zillaslab, Palatino, Palatino Linotype, x-locale-heading-secondary, serif"><span style="background-color: #333333;">使用</span></font></code>async函数重写 promise 链</h3>
+<h3 id="使用async函数重写_promise_链"><code><font face="x-locale-heading-primary, zillaslab, Palatino, Palatino Linotype, x-locale-heading-secondary, serif">使用</font></code>async函数重写 promise 链</h3>
```

Comment:
Please remove it along with `code` and `font face` markup. Thx

Reply:
@t7yang  ok.